openwisp_notifications/tasks.py

```diff
@@ -202,3 +210,67 @@ def delete_ignore_object_notification(instance_id):
     Deletes IgnoreObjectNotification object post it's expiration.
     """
     IgnoreObjectNotification.objects.filter(id=instance_id).delete()
+
+
+@shared_task(base=OpenwispCeleryTask)
+def batch_email_notification(instance_id):
```

Comment:
I am not convinced of the name of this function here.

Does this function batch emails, or does it send batched emails? From the name it seems it does the former, but from the code it seems to me it does the latter. Please clarify.

If it's the latter, we should rename it to something like `send_batched_email_notifications` or something similar.